index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header>
+        <div class="main-container">
+            <div class="header-container">
+                <div class="my-logo">
+                  <h1 class="logo">My Logo</h1>
+                </div>
+                <div class="hambuger">
+                  <div class="bar"></div>
+                  <div class="bar"></div>
+                  <div class="bar"></div>
+                </div>
+            </div>
+        </div>
+    </header>
+    <section class="img-BG">
+        <div class="main-container">
```

Comment:
- [ ] [REQUIRED] Kindly follow the instructions in the image below:

| Your Design |
|-------------|
|![image](https://user-images.githubusercontent.com/62938735/191797144-8689638e-3b4d-4d63-9a2e-691ba131f444.png)|